user/environment-variables.md

```diff
@@ -162,8 +162,8 @@ The following default environment variables are available to all builds.
 - `CONTINUOUS_INTEGRATION=true`
 - `DEBIAN_FRONTEND=noninteractive`
 - `HAS_JOSH_K_SEAL_OF_APPROVAL=true`
-- `USER=travis` (**do not depend on this value**; do not override this value)
-- `HOME=/home/travis` (**do not depend on this value**)
+- `USER=travis`
+- `HOME` is set to the expected value depending on the platform.
```

Comment:
@plaindocs do you think this is self-explanatory? 
Right now, values are: 

- mac OS:  `Users/travis`
- windows:  `/c/Users/travis`
- linux:  `/home/travis`

Reply:
I'd prefer to make it explicit, and list the three cases.